comictaggerlib/seriesselectionwindow.py

```diff
@@ -465,24 +465,24 @@ def categorize(result: ComicSeries) -> int:
             self.twList.setItem(row, 0, item)
 
             item_text = str(record.start_year)
-            item = QtWidgets.QTableWidgetItem(item_text)
+            item = QtWidgets.QTableWidgetItem(record.start_year)
             item.setData(QtCore.Qt.ItemDataRole.ToolTipRole, item_text)
+            item.setData(QtCore.Qt.ItemDataRole.DisplayRole, record.start_year)
             item.setFlags(QtCore.Qt.ItemFlag.ItemIsSelectable | QtCore.Qt.ItemFlag.ItemIsEnabled)
             self.twList.setItem(row, 1, item)
 
             item_text = str(record.count_of_issues)
-            item = QtWidgets.QTableWidgetItem(item_text)
+            item = QtWidgets.QTableWidgetItem(record.count_of_issues)
             item.setData(QtCore.Qt.ItemDataRole.ToolTipRole, item_text)
             item.setData(QtCore.Qt.ItemDataRole.DisplayRole, record.count_of_issues)
             item.setFlags(QtCore.Qt.ItemFlag.ItemIsSelectable | QtCore.Qt.ItemFlag.ItemIsEnabled)
             self.twList.setItem(row, 2, item)
 
-            if record.publisher is not None:
```

Comment:
Unless you can prove that record.publisher is never None this needs to stay there are several other checks to see if the publisher is None in other parts of the code and None of them set it to the empty string.

Reply:
If it's `None` then the result is an empty field. It doesn't make sense to do the work though so I'll revert it. I'll also add the same check for `year` and `count_of_issues` as they can be empty too.

Reply:
I don't remember where exactly but CT has had issues with None as it does not sort with other types `<` or `>` operators in python